spec/yaml/tsg_lmc_contactdb_v3.yaml

```diff
@@ -2294,9 +2259,7 @@ paths:
                     - field: null
                       message: authorization required
         '404':
-          description: '"segment_id" : "Returned if segment_id is not valid"
-
-            "segment_id" : "Returned if segment_id does not exist"'
+          description: The segment_id is not valid or does not exist
```

Comment:
segment_id -> `segment_id`

Reply:
I think we'd need to make this and related fixes in the source repo, otherwise fixing it here would just be overwritten in the next librarian release? 🤔